src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngineResolver.cs

```diff
@@ -11,11 +14,18 @@ namespace Microsoft.PowerFx
     /// <summary>
     /// <see cref="INameResolver"/> implementation for <see cref="RecalcEngine"/>.
     /// </summary>
-    internal class RecalcEngineResolver : SimpleResolver
+    internal class RecalcEngineResolver : SimpleResolver, INameResolver2
     {
         private readonly RecalcEngine _parent;
         private readonly PowerFxConfig _powerFxConfig;
 
+        public IReadOnlyDictionary<string, NameLookupInfo> GlobalSymbols => _parent.Formulas.ToDictionary(kvp => kvp.Key, kvp => Create(kvp.Key, kvp.Value));
+
+        private NameLookupInfo Create(string name, RecalcFormulaInfo recalcFormulaInfo)
+        {
+            return new NameLookupInfo(BindKind.PowerFxResolvedObject, recalcFormulaInfo.Value.Type._type, DPath.Root, 0, recalcFormulaInfo, displayName: new DName($"{name} variable"));
```

Comment:
`displayName` in a NameLookupInfo is used for providing localized/human-readable names for symbols, i.e. the Dataverse cr425b_user has a display name "User" if we're working in en-US. 

Is "{name} variable" the user-visible display name we want for variables? If so, please add a test to DisplayNameTests covering that.

Reply:
Removed